src/EditorFeatures/CSharpTest/Formatting/Indentation/FormatterTestsBase.cs

```diff
@@ -183,6 +191,35 @@ public static void TestIndentation(int indentationLine, int? expectedIndentation
             var actualIndentation = provider.GetDesiredIndentation(indentationLineFromBuffer);
 
             Assert.Equal(expectedIndentation, actualIndentation);
+
+            TestBlankLineIndentationService(
+                workspace, indentationLine, expectedIndentation, textView.Object);
+        }
+
+        private static void TestBlankLineIndentationService(
+            TestWorkspace workspace, int indentationLine, int? expectedIndentation, ITextView textView)
+        {
+            var snapshot = workspace.Documents.First().TextBuffer.CurrentSnapshot;
+            var indentationLineFromBuffer = snapshot.GetLineFromLineNumber(indentationLine);
+
+            var document = workspace.CurrentSolution.Projects.Single().Documents.Single();
+            var blankLineIndenter = (IBlankLineIndentationService)document.GetLanguageService<ISynchronousIndentationService>();
+            var indentStyle = workspace.Options.GetOption(FormattingOptions.SmartIndent, LanguageNames.CSharp);
+            var blankLineIndentResult = blankLineIndenter.GetBlankLineIndentation(
+                document, indentationLine, indentStyle, CancellationToken.None);
+
+            var blankLineIndentation = blankLineIndentResult.GetIndentation(textView, indentationLineFromBuffer);
+            if (expectedIndentation == null)
+            {
+                if (indentStyle == IndentStyle.None)
```

Comment:
💭 Maybe pass in `expectedBlankLineIndentation` so it's always tested? Right now we're getting coverage but it's not looking like coverage when you see just the individual tests.

Reply:
Yes.  That's much cleaner.   Have done ths.